src/NodeServices/IAnalyticsClient.cs

```diff
@@ -433,7 +433,27 @@ public enum Actions
         /// <summary>
         /// ExportSettings event, e.g. tracks the ExportSettings event
         /// </summary>
-        ExportSettings
+        ExportSettings,
+
+        /// <summary>
+        /// Ajusting the confidence level threshold in the Preferences panel
+        /// </summary>
+        AdjustConfidenceLevel,
+
+        /// <summary>
+        /// Click in the Low Confidence Level option will open the list of possible nodes
+        /// </summary>
+        ExpandConfidenceLevel,
+
+        /// <summary>
+        /// The user selected Object Type then it will affect the node autocomplete results
+        /// </summary>
+        SelectObjectType,
```

Comment:
Let's re-use `Select` and specify the option as additional info

Reply:
This action was deleted and instead we are using the Select option, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13581/commits/0655f6d242fa449dc828a75d8634567d3b5737f4